Recruit-Training/Advanced-Recruit-Training/Telemetry-Teaser/src/App.tsx

```diff
@@ -1,26 +1,63 @@
+import { useState } from "react";
+
 import BatteryInput from "~/components/batteryInput";
 import Header from "~/components/header";
 import SpeedInput from "~/components/speedInput";
 import WeatherInput from "~/components/weatherInput";
 
 const App = () => {
-  const calculateRange = () => {
-    return;
-  }
+  const [s, setSpeed] = useState(0);
+  const [b, setBattery] = useState(0);
+  const [w, setWeather] = useState(0);
+  const range = -((s * s * b) / 2500) + 4 * b + w;
+  const [output, setOutput] = useState<string>("");
+
+  const calculateRange = (range: string) => {
```

Comment:
With this logic, it's only showing one condition at a time.
If I set speed to 10000, and leave the battery percentage blank, I only see `Enter a Speed between 0km/h and 90km/h`, when I should be seeing both `Enter a Speed between 0km/h and 90km/h` and `Enter Value for Battery %`. This is true for all conditions where you need to display multiple error messages at once.